soh/include/variables.h

```diff
@@ -166,6 +166,8 @@ extern "C"
 	extern s32 gSystemArenaLogSeverity;
 	extern u8 __osPfsInodeCacheBank;
 	extern s32 __osPfsLastChannel;
+	extern u8 gSpeedToggle1;
+	extern u8 gSpeedToggle2;
```

Comment:
might be nice to have these names match the `gEnableWalkModify`/`gWalkModifierOne` naming convention, possibly `gWalkModifyToggle`?

i'd be happy with any name that specifies that it's *walk* speed being modified